Woof/Woof/View/SitterFlow/SitterProfileView/SitterInfoCardView.swift

```diff
@@ -0,0 +1,109 @@
+import SwiftUI
+
+struct SitterInfoCardView: View {
+    // MARK: - Internal interface
+
+    /// The name of the pet sitter.
+    let name: String
+
+    /// The surname of the pet sitter.
+    let surname: String
+
+    /// The phone of the pet sitter.
+    let phone: String
+
+    /// The brief information about the pet sitter.
+    let bio: String
+
+    /// The price charged by the pet sitter per hour for walking.
+    let ratePerHour: Double
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: AppStyle.UIElementConstant.spacingBetweenElements) {
+            HStack(alignment: .center) {
+                VStack(alignment: .leading, spacing: AppStyle.UIElementConstant.spacingBetweenElements) {
+                    HStack {
+                        Text(name)
+                            .lineLimit(lineLimitForNameAndSurname)
+                        Text(surname)
+                            .lineLimit(lineLimitForNameAndSurname)
+                    }
```

Comment:
I think we can apply a modifier for the stack, not for every element to avoid code duplication
```suggestion
                    HStack {
                        Text(name)
                        Text(surname)                       
                    } .lineLimit(lineLimitForNameAndSurname)
```

Reply:
Check the new version.